Makefile

```diff
@@ -64,7 +60,7 @@ cluster-sync:
 
 .PHONY: cluster-functest
 cluster-functest:
-	scripts/kubevirtci.sh functest
```

Comment:
Can you drop the `functest` func from `scripts/kubevirtci.sh` and `scripts/kubevirt.sh` too please?